src/types.ts

```diff
@@ -1,55 +1,37 @@
 export interface Creator {
   'name': string,
   'id': number
-}
+};
+
+export interface CreatorPost {
+  "creator_id": number
+};
 
 export interface User {
   'name': string,
   'follows': Creator[]
-}
+};
 
 export interface YoutubeVideo {
-  id: string;
-  image: string;
-  publishedAt: string;
-  title: string;
-}
+  id: string,
```

Comment:
Technically, the interface syntax needs to be changed to be ; instead of , . Both work working, just proper format to use ;